modules/react/src/index.ts

```diff
@@ -5,6 +5,11 @@
 export {default as DeckGL} from './deckgl';
 export {default} from './deckgl';
 
+// Widgets
+export {CompassWidget} from './widgets/compass-widget';
```

Comment:
How come we don't have to explicitly export all the Layers supported in JSX, but we have to do so with widgets? Can we not use the same [masquerading trick](https://github.com/visgl/deck.gl/blob/48b19ab9044dcff2a9c35e3c7c36c517614932b5/modules/react/src/deckgl.ts#L122-L126)?

Reply:
This was modeled after react-map-gl's [implementation](https://github.com/visgl/react-map-gl/blob/master/src/components/attribution-control.ts). 

We had a use case in mind for folks who want to implement their own widget UI in react like in this [commit](https://github.com/visgl/deck.gl/commit/271561f6877e0d0b925a13833cc9e1fe3deb247c).
<img width="653" alt="Screenshot 2024-12-12 at 5 59 38 PM" src="https://github.com/user-attachments/assets/9ef6fcea-25c7-4c5d-ac5a-9ecaaede68ba" />